librosa/filters.py

```diff
@@ -599,7 +599,7 @@ def constant_q(sr, fmin=None, n_bins=84, bins_per_octave=12, tuning=0.0,
     if pad_fft:
         max_len = int(2.0**(np.ceil(np.log2(max_len))))
     else:
-        max_len = np.ceil(max_len)
+        max_len = int(np.ceil(max_len))
```

Comment:
I think this cast might be better implemented from within `util.pad_center` instead of in the calling function.